kpi/tests/api/v2/test_api_asset_permission_assignment.py

```diff
@@ -938,3 +940,48 @@ def test_partial_permission_invalid(self):
         )
         response = self.client.post(bulk_endpoint, assignments, format='json')
         assert response.status_code == status.HTTP_400_BAD_REQUEST
+
+    def test_bulk_assign_perm_reduces_queries(self):
+        permissions = [PERM_VIEW_ASSET, PERM_CHANGE_ASSET]
+
+        # Assign permissions one by one
+        with CaptureQueriesContext(connection) as old_context:
+            for user in [self.someuser, self.anotheruser]:
+                for perm in permissions:
+                    self.asset.assign_perm(user, perm)
+        old_method_query_count = len(old_context.captured_queries)
+
+        # Bulk assign permissions
+        with CaptureQueriesContext(connection) as new_context:
```

Comment:
We should test this with an assertNumQueries so we'll know if something makes the numbers creep up again